checks/zk_quorum.go

```diff
@@ -0,0 +1,129 @@
+package checks
+
+import (
+    "bufio"
+    "fmt"
+	"time"
```

Comment:
Looks like some indentation issues here. All code should be run through `gofmt`.